### PR TITLE
[set-theory/zh-cn] Add Chinese translation to set-theory.html.md 

### PR DESCRIPTION
**Pull request by lo0b0o**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zh-cn/set-theory-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+* 集合由不重复的项组成。
+
+## 基本符号
+
+### 运算符
+* 并运算符，`∪`，表示“或”；
+* 交运算符，`∩`，表示“且”；
+* 差运算符，`\`，表示“不包括”；
+* 补运算符， `'`，表示补集;
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:20]:
```suggestion
* 补运算符，`'`，表示补集;
```

Comment (inkydragon) [reply zh-cn/set-theory-cn.html.markdown:20]: 中文标点自带空格，就无需手动加空格了。
另外也是保持行文统一

File: zh-cn/set-theory-cn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+* 叉积运算符，`×`，表示笛卡尔积。
+
+### 限定词
+* 冒号限定词， `:`，表示“使得”；
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:24]:
```suggestion
* 冒号限定词，`:`，表示“使得”；
```

File: zh-cn/set-theory-cn.html.markdown [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+* 从属限定词，`∈`，表示“属于”；
+* 子集限定词， `⊆`, 表示“是……的子集”;
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:26]:
```suggestion
* 子集限定词，`⊆`，表示“是……的子集”;
```

File: zh-cn/set-theory-cn.html.markdown [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+* 真子集限定词， `⊂`，表示“是……的真子集”。
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:27]:
```suggestion
* 真子集限定词，`⊂`，表示“是……的真子集”。
```

File: zh-cn/set-theory-cn.html.markdown [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+
+###  重要的集合
+* `∅`， 空集，即不包含任何元素的集合；
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:31]:
```suggestion
* `∅`，空集，即不包含任何元素的集合；
```

File: zh-cn/set-theory-cn.html.markdown [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+集合的基数，或者说大小，由该集合中的项目数量决定。基数运算符为 `|...|`。
+
+例如，若 `S = { 1, 2, 4 }`，则 `|S| = 3`。
+
+### 空集
+  
+* 可以在集合符号中使用不成立的条件来构造空集，例如，`∅ = { x : x ≠ x }`，或 `∅ = { x : x ∈ N, x < 0 }`；
+* 空集总是唯一的（即，有且只有一个空集）；
+* 空集是所有集合的子集；
+* 空集的基数为 0，即， `|∅| = 0`。
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:52]:
```suggestion
* 空集的基数为 0，即 `|∅| = 0`。
```

File: zh-cn/set-theory-cn.html.markdown [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+集合构造器符号是构造集合的一种更具描述性的方式。它依赖于一个主语和一个谓词，使得 `S = { 主语 : 谓词 }`。 例如，
+
+```
+A = { x : x 是元音字母 } = { a, e, i, o, u, y}
+B = { x : x ∈ N, x < 10 } = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 }
+C = { x : x = 2k, k ∈ N } = { 0, 2, 4, 6, 8, ... }
+```
+
+有时，谓语可能会 "漏 "到主语中，例如，
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:72]:
```suggestion
有时，谓词可能会 "漏 "到主语中，例如，
```

File: zh-cn/set-theory-cn.html.markdown [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+* 如果值 `a` 包含在集合 `A` 中，那么我们说 `a` 属于 `A`，并用符号表示为 `a ∈ A`。
+* 如果值 `a` 不包含于集合 `A` 中，那么我们说 `a` 不属于 `A`，并用符号表示为 `a ∉ A`。
+
+### 相等关系
+
+* 如果两个集合包括相同的项，那么我们说这两个集合相等，例如，`A = B`。
+* 集合的相等关系于顺序无关，例如， `{ 1, 2, 3, 4 } = { 2, 3, 1, 4 }`。
+* 集合中的元素不能重复, e.g. `{ 1, 2, 2, 3, 4, 3, 4, 2 } = { 1, 2, 3, 4 }`。
+* 集合 `A` 与 `B` 相等当且仅当`A ⊆ B` 且 `B ⊆ A`。
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:90]:
```suggestion
* 集合 `A` 与 `B` 相等当且仅当 `A ⊆ B` 且 `B ⊆ A`。
```

File: zh-cn/set-theory-cn.html.markdown [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+### 从属关系
+
+* 如果值 `a` 包含在集合 `A` 中，那么我们说 `a` 属于 `A`，并用符号表示为 `a ∈ A`。
+* 如果值 `a` 不包含于集合 `A` 中，那么我们说 `a` 不属于 `A`，并用符号表示为 `a ∉ A`。
+
+### 相等关系
+
+* 如果两个集合包括相同的项，那么我们说这两个集合相等，例如，`A = B`。
+* 集合的相等关系于顺序无关，例如， `{ 1, 2, 3, 4 } = { 2, 3, 1, 4 }`。
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:88]:
```suggestion
* 集合的相等关系于顺序无关，例如 `{ 1, 2, 3, 4 } = { 2, 3, 1, 4 }`。
```

File: zh-cn/set-theory-cn.html.markdown [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+* 集合中的元素不能重复, e.g. `{ 1, 2, 2, 3, 4, 3, 4, 2 } = { 1, 2, 3, 4 }`。
```
Comment (inkydragon) [inline zh-cn/set-theory-cn.html.markdown:89]:
```suggestion
* 集合中的元素不能重复，例如 `{ 1, 2, 2, 3, 4, 3, 4, 2 } = { 1, 2, 3, 4 }`。
```

**Review by inkydragon — APPROVED** [review]
Some format Suggestions.

**Comment by imba-tjd** [comment]
Thanks @lo0b0o @inkydragon

**Pull request merged**